Split element templates docs into C7 and C8 (#2369)
camunda/camunda-platform-docs: docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

The example defines five custom fields, each mapped to different technical properties:

- The task type `http` is mapped to the `zeebe:taskDefinition:type` property in BPMN 2.0 XML
- The `REST Endpoint URL` and `REST Method` are mapped to `task headers`
- The `Request Body` is mapped to a local variable via an `input parameter`
- The `Result Variable` is mapped into a process variable via an `output parameter`

> The task type is hidden to the user, and other properties are editable by the user through the properties panel as shown in the following screenshot:

Reviewer: This sentence is slightly confusing. The fact of the matter is that _only_ the properties specified as part of the template are editable. No other properties are editable by default.

Author: How about:

```suggestion
The task type is hidden to the user, and other properties specified in the template are editable by the user through the properties panel as shown in the following screenshot:
```

Reviewer: ```suggestion
The task type is hidden to the user. Properties specified in the template can be edited through the properties panel as shown in the following screenshot:
```

###### Supported types

The property `feel` is supported on the following input types:

- `String`
- `Text`

> #### Bindings

Reviewer: I can't find the binding type `bpmn:Message#property` documented anywhere. Are we aware of that?

Author: Hmm I believe I overlooked this when I implemented the feature 🤡

##### Supported Bindings

> The following binding types can be `optional`:

Author: OK

- `zeebe:input`
- `zeebe:output`
- `zeebe:taskHeader`

> - `zeebe:property`

Reviewer: I think we forgot about `bpmn:Message#zeebe:subscription#property` here (cf. https://github.com/bpmn-io/bpmn-js-element-templates/blob/main/src/cloud-element-templates/create/MessageZeebeSubscriptionBindingProvider.js#L39).

Author: Cf. https://github.com/camunda/camunda-platform-docs/pull/2369#discussion_r1283132249
